package-lock.json

```diff
@@ -5556,30 +5556,6 @@
       "integrity": "sha512-jf84uxzwiuiIVKiOLpfYk7N46TSy8ubTonmneY9vrpHNAnp0QBt2BxWV9dO3/j+BoVAb+a5G6YDPW3M5HOdMWQ==",
       "dev": true
     },
-    "mini-css-extract-plugin": {
```

Comment:
Why this change?

Reply:
Package lock was updated to reflect commit b7f90c265f530ee00bc12d9d6d0ce0fa6d4d9861 removing the plugin from package.json.